src/sql/src/func.rs

```diff
@@ -2571,6 +2571,9 @@ pub static PG_CATALOG_BUILTINS: Lazy<BTreeMap<&'static str, Func>> = Lazy::new(|
         "upper_inf" => Scalar {
             params!(RangeAny) => UnaryFunc::RangeUpperInf(func::RangeUpperInf) => Bool, 3854;
         },
+        "uuid_generate_v5" => Scalar {
+            params!(Uuid, String) => BinaryFunc::UuidGenerateV5, 16405;
```

Comment:
16405 is not the correct oid. Do you have a postgres installed with this extension enabled? I don't, otherwise I'd figure it out. `select oid from pg_proc where proname = 'uuid_generate_v5'` would print it.

Reply:
hmm, that's what I've got here unless I'm missing something? 
![image](https://user-images.githubusercontent.com/124720264/221299438-75a89c89-3acc-4537-b660-74eb4d256968.png)

Reply:
! oh I don't think we've encountered this before. In that case, add the next oid to oid.rs I think? @sploiselle thoughts?

```
pub const FUNC_PG_UUID_GENERATE_V5: u32 = 16_565;
```

Reply:
Yeah, this is just in the range of OIDs that postgres doesn't reserve. When you install the extension it likely just chooses the next _n_ available OIDs, which is in the same range as we use for _our_ OIDs. Said another way, this OID isn't fixed in PG; you could contrive a situation that results in this function having a different OID. I'd say just choose the next OID available as Matt suggests and looks like you've already done.